script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_10.txt

```diff
@@ -3,44 +3,44 @@
 <Page2>
 Without anyone waking me up,
 Without hearing anything,
-Vaguely without any reason,# I wonder when was the last time I woke up in the middle of my sleep.
+Seemingly without any reason,# I wonder when the last time was that I woke up in the middle of my sleep.
 <Page3>
-I feel like I have forgotten something, so I sit up restlessly.
+I feel like I've forgotten something, so I sit up restlessly.
 <Page4>
-I feel my heart pounding, so I leave the room, curious about what is going on outside.
+I feel my heart pounding, so I leave the room, curious about what's going on outside.
 <Page5>
-I am thirsty, so I go down the hallway in search of water.
+I'm thirsty, so I go down the hallway in search of water.
 <Page6>
 A long, moonlit corridor.
 The sounds of the night after the inhabitants have fallen asleep.
 ...How nostalgic.
 I'm sure I've walked down a path like this before.
-As though wandering lost, paying no heed to my surroundings, I begin to climb the stairs.
+As though wandering in a daze, paying no heed to my surroundings, I begin to climb the stairs.
 <Page7>
 It is a blind spot in the mansion.
-The staircase not only connects the first floor to the second floor but on the other side of the staircase, there is a secret passage that leads to the attic.
+The staircase not only connects the first floor to the second, but on the other side, there is a secret passage that leads to the attic.
 <Page8>
 It has long since been forgotten, even by the residents.
 The attic of the mansion, once used as a storehouse, has not been accessed by anyone for several years.
 <Page9>
-Even though it is October, it is as cold as the middle of winter tonight.
+Despite only being October, tonight is as cold as the middle of winter.
 I search through the luggage for a blanket.
 An old cabinet catches my attention, so I open a worn-out drawer to look inside.
 <Page10>
 "――――――"
-There is nothing but a bunch of small, insignificant things inside.
+There is nothing but small, insignificant things inside.
 The contents of the open drawer rattle about.
 <Page11>
-The feeling of disappointment, as if I opened a treasure chest only to find a few pebbles scattered inside.
+I feel disappointed, as if I opened a treasure chest only to find a few pebbles scattered inside.
 There are badges, accessories, binoculars, and other miscellaneous items collected by children lying around.
-Among them, lies one item that clearly stands apart from the others.
+Among them lies one item that clearly stands apart from the others.
 <Page12>
 It is a dirty bandage stored in a transparent plastic case.
-It is neatly rolled up, but it's too old to be of any use.
-There's nothing here.
+It's neatly rolled up, but too old to be of any use.
+There's nothing else here.
```

Comment:
I think this alters the translation too much - 
> ここには何もない。

I think the intent is 'there's nothing here [of importance]'.